addons/binding/org.openhab.binding.helios/ESH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,173 @@
+        <item-type>String</item-type>
+        <label>Device State</label>
+        <description>State of the Helios device</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="devicestatestamp">
+        <item-type>DateTime</item-type>
+        <label>Time of state of the device</label>
+        <description>Event date and time in CCYY-MM-DDThh:mm:ss format</description>
+        <state readOnly="true" />
+    </channel-type>
+        <channel-type id="audiolooptest" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Audio Loop Test</label>
+        <description>Indicates the result of the audio loop test</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="audioloopteststamp">
+        <item-type>DateTime</item-type>
+        <label>Time the audio loop test</label>
+        <description>Event date and time in CCYY-MM-DDThh:mm:ss format</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="motion">
+        <item-type>Switch</item-type>
+        <label>Motion</label>
+        <description>Indicates the detection of motion</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="motionstamp">
+        <item-type>DateTime</item-type>
+        <label>Time when motion was detected</label>
+        <description>Event date and time in CCYY-MM-DDThh:mm:ss format</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="noise">
+        <item-type>Switch</item-type>
+        <label>Noise</label>
+        <description>Indicates the detection of noise</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="noisestamp">
+        <item-type>DateTime</item-type>
+        <label>Time when noise was detected</label>
+        <description>Event date and time in CCYY-MM-DDThh:mm:ss format</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="keyreleased">
+        <item-type>String</item-type>
+        <label>Key Released</label>
+        <description>Code of the key released</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="keyreleasedstamp">
+        <item-type>DateTime</item-type>
+        <label>Time a key was released</label>
+        <description>Event date and time in CCYY-MM-DDThh:mm:ss format</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="switchenabler" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Switch Enabler</label>
+        <description>Activate or deactivate a switch in the Helios device</description>
+    </channel-type>
+    <channel-type id="trigger" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Switch Trigger</label>
+        <description>Trigger the state of a switch in the Helios device</description>
```

Comment:
How can one trigger a state? I have no clue what this description tries to explain to me 😕